atom/browser/atom_javascript_dialog_manager.cc

```diff
@@ -52,9 +52,14 @@ void AtomJavaScriptDialogManager::RunJavaScriptDialog(
     return;
   }
 
+  int default_id = -1;
```

Comment:
can you include a comment here for future file visitors about what these id values do?